processing/fb_event_processor.coffee

```diff
@@ -54,9 +55,10 @@ exports.extractAndSendEvents = (feed) ->
             location: getLocationInfo FBevent
             isAllDay: FBevent.end_time?
             startTime: startTime
-            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else null
+            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else 0
```

Comment:
Are you sure an end time of 0 as opposed to null is a good idea? I assume these are for events with no known endtime. 0 is a specific time (the epoch) and seems to me to be misleading. `null` actually means that there is none. Seems like you would need special case handling for the 0. How would you distinguish 0 for no-end-time from 0 for ending at 1970-01-01T00:00:00Z?